addons/test_discuss_full/tests/test_performance.py

```diff
@@ -816,6 +816,7 @@ def _get_init_messaging_result(self):
                                         'active': True,
                                         'country': [('clear',)],
                                         'id': self.users[0].partner_id.id,
+                                        'is_bot': False,
```

Comment:
I don't understand this change, it look like a bad rebase?

Reply:
It's a move of the code that was incorrectly put in livechat and made the query count not correctly applied.